src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -33,7 +34,7 @@ class FireCloudServiceActor extends HttpServiceActor with FireCloudDirectives
     def actorRefFactory = context
   }
 
-  val agoraDAO:AgoraDAO = new HttpAgoraDAO(FireCloudConfig.Agora.authUrl)
+  val agoraDAO:AgoraDAO = new HttpAgoraDAO(FireCloudConfig.Agora.authUrl, FireCloudConfig.Agora.baseUrl)
```

Comment:
Suggestion: Instead of adding a second url to the constructor, change the constructor to take the whole Agora config. Then, inside HttpAgoraDAO, do whatever you need to do with the different urls. This frees us up from expanding the constructor and opens up some freedom inside the dao class when we need to add more functionality.